### PR TITLE
Finished PlayerTest off

### PR DESCRIPTION
**Pull request by Mikeywalsh**
Finished testing all of Player in PlayerTest. Almost all code is covered, a small amount is left uncovered as it depends on LibGDX or is completely unnessessary to test.

Coverage:
96% of Methods
97% of Lines

### CODE REVIEW

File: core/src/io/github/teamfractal/entity/Player.java [hunk 1]
```diff
@@ -257,28 +257,21 @@ public void sellResourceToMarket(int amount, Market market, ResourceType resourc
      * @param plot           The landplot to purchase
      */
```
Comment (RobinStephenson) [inline core/src/io/github/teamfractal/entity/Player.java]: Coulp add throws documenation for the NotEnoughMoneyException. Dont need to both with null pointer exception as thats pretty obvious

**Review by RobinStephenson — APPROVED** [review]
just needs one small docstring update. Good otherwise so im approving so you can just merge after adding the comment

File: core/src/io/github/teamfractal/entity/Player.java [hunk 2]
```diff
@@ -255,30 +255,24 @@ public void sellResourceToMarket(int amount, Market market, ResourceType resourc
     /**
      * Player add a landplot to their inventory for gold
      * @param plot           The landplot to purchase
+     * @throws NotEnoughMoneyException if the player does not have enough money for the tile
```
Comment (Mart500) [inline core/src/io/github/teamfractal/entity/Player.java:258]: Add an Extra @throws for PlotAleadyOwnedException

**Review by Mart500 — COMMENTED** [review]
@Mikeywalsh Small doc string change needed.

**Pull request merged**